docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       RAILS_LOG_TO_STDOUT: "1"
       DATABASE_SSLMODE: disable
       DISABLE_HTTPS: "1"
+      BC_USE_DEV_MOCK: "true"
```

Comment:
Not really related to this work, but without this, the local docker image can't pass the DWP check so we can't do end-to-end tests using docker-compose. This has no effect on real envs, it is only for local machines.